### PR TITLE
Includes documentation for tracking links and a new JSON Schema section

### PR DESCRIPTION
**Pull request by richeyb**

### CODE REVIEW

File: source/schemas/tracking_links.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    },
+    "token": {
+      "description": "The token included in the job board link.",
+      "title": "Token",
+      "type": "string"
+    },
+    "related_post_id": {
+      "description": "The ID of the related post or email for this job token",
+      "title": "Related Post ID",
+      "type": ["integer", "null"]
```
Comment (jaydeesimon) [inline source/schemas/tracking_links.json]: Interesting that a nullable type is denoted by using the string `"null"`. I wonder why they decided on that and not just using the value null?

Comment (richeyb) [inline source/schemas/tracking_links.json]: Because it describes a type, not a value. If the value were `null` it could get stripped out

Comment (jaydeesimon) [inline source/schemas/tracking_links.json]: Makes sense.

**Comment by amandasegal** [comment]
Hey @richeyb , is it okay if I update the description of what Tracking Links are? It's a bit more in layman's terms, but often the developers we work with don't see how the links are generated. I was also thinking that the name of the endpoint could be "Tracking Link Data for Token". In support we often use the term "Tracking Link" to describe the actual URL, I think "Tracking Link for Token" might lead people to think that the endpoint actually generates the URL. 

Tracking Links

Tracking Links are URLs which allow users to track the source and/or referrer of candidates who applied through the link. Greenhouse generates Tracking Links by appending a token to the end of a job post or job boad URL. These tokens represent a combination of source_id and/or referrer_id, and can link back to any combination of job posts, job boards, and jobs. 


Tracking Link Data for Token

Retrieve the specific tracking link data for the supplied token.

**Comment by Diamond** [comment]
@amandasegal Yeah, I'm all for that! I appreciate the help with the wording; I'm sometimes too close to the development of things to be able to figure out the right way to word things! 👍
